dissect/target/helpers/fsutil.py

```diff
@@ -561,6 +562,29 @@ def open_decompress(
     return file
 
 
+def reverse_read(fh: io.BytesIO, chunk_size: int = 1024 * 10, reverse_chunk: bool = True) -> Iterator[bytes]:
+    """Like iterating over a ``BytesIO`` file-like object, but starting from the end of the file.
+
+    Args:
+        fh: The file-like object (opened in binary mode) to read from.
+        chunk_size: The chunk size to use for iterating over bytes (default: 10KB).
+        reverse_chunk: Whether we should reverse the bytes of each read chunk (default: True).
```

Comment:
```suggestion
        reverse_chunk: Whether we should reverse the bytes of each chunk (default: True).
```
Doesn't Sphinx also list the default arguments already?